projects/core/src/lazy-loading/tokens.ts

```diff
@@ -0,0 +1,13 @@
+import { InjectionToken } from '@angular/core';
+
+/**
+ * The MODULE_INITIALIZER is used as a multi provider that returns
+ * a function that should be executed when the module is lazy loaded.
+ * It is, in a way, an APP_INITIALIZER for lazy loaded modules.
+ *
+ * If the module is eagerly loaded, the MODULE_INITIALIZER functions
```

Comment:
so we can maybe add, that they will work like `APP_INITIALIZER'`s

Reply:
That would be helpful indeed. I added:

```
Therfore, if the module in which it is defined is used in a eager loading configuration, MODULE_INITIALIZER will fall back to work like APP_INITIALIZER.
```